include/cantera/kinetics/RxnRates.h

```diff
@@ -106,6 +106,97 @@ class Arrhenius
     doublereal m_logA, m_b, m_E, m_A;
 };
 
+//! An Arrhenius-like rate type which depends on both gas and electron temperature
+/**
+ * A reaction rate coefficient of the following form.
+ *
+ *   \f[
+ *        k_f =  A Te^b \exp (-E/RT) \exp(-EE/RTe),
+ *   \f]
+ * where T and Te are the gas and electron temperature, respectively.
+ */
+class ElectronTemperature
```

Comment:
In #1101, there is a new `ArrheniusBase` class defined in `Arrhenius.h`, which would be a convenient base class for this (and avoid some duplicate method definitions). Also, would there be a more descriptive name?